### DISCUSSION TITLE
Show a zero-width joined characters

### DISCUSSION THREAD

**Discussion by ray1997** [character-map-uwp/Character-Map-UWP#269] [Ideas]
There were lots of characters that not show on an emoji list (Segoe UI Emoji) as it was a joint of two emoji.
It might be optional to show those combos, but a nice addition to have on a glyph list and able to export them.

**Comment by JohnnyWestlake** [comment]
The quick way of doing this would be to integrate https://github.com/neosmart/unicode.net - which works fairly smoothly and easily.
![image](https://user-images.githubusercontent.com/2319473/78892085-9f858c00-7a60-11ea-89cf-a5be77a5f9dc.png)

Buuuut from my point of this lib adds too much of RAM and startup cost so will work on a more bespoke solution at some point. Probably involving SQLite because search should support it too

Will need a button to also switch between the actual character map and the full emoji map (and hence a way of detecting if a font contains emoji)

**Comment by JohnnyWestlake** [comment]
Tracked on [/Character-Map-UWP/EmojiMap](https://github.com/EdiWang/Character-Map-UWP/tree/emojimap) branch and this project: https://github.com/EdiWang/Character-Map-UWP/projects/1

Still a number of issues to solve - the big blocker being trying to figure out what emoji combinations a font file actually supports (DirectWrite falls back to Segoe UI Emoji automatically which isn't massively helpful for this case 😅)
